crates/bevy_transform/src/systems.rs

```diff
@@ -320,7 +320,7 @@ mod test {
 
         // Adding the system in a single threaded stage. As the system will panic, this will
         // only bring down the current test thread.
-        app.add_stage("single", SystemStage::single_threaded())
+        app.add_stage("single", SystemStage::parallel())
             .add_system_to_stage("single", transform_propagate_system);
```

Comment:
can probably change this back to
```rust
app.add_system(transform_propagate_system);
```
as it was before https://github.com/bevyengine/bevy/pull/6172/files

Reply:
Done.